apps/wallet-mobile/src/features/Swap/common/helpers.ts

```diff
@@ -65,3 +66,28 @@ export const parseOrderTxMetadata = (metadataJson: string): OrderTxMetadata | nu
   const parsedMetadata = parseSafe(metadataJson)
   return isOrderTxMetadata(parsedMetadata) ? parsedMetadata : null
 }
+
+function containsOnlyValidChars(str?: string): boolean {
```

Comment:
@SorinC6 can you check `/src/utils/sorting.ts` there is a helper sortTokenInfos, I think it can do the job.